RELEASE_NOTES.rst

```diff
@@ -21,6 +21,119 @@
 
 .. towncrier release notes start
 
+Airflow 2.5.1 (2023-01-20)
+--------------------------
+
+Significant Changes
+^^^^^^^^^^^^^^^^^^^
+
+Trigger gevent ``monkeypatching`` via environment variable (#28283)
```

Comment:
You also should delete the newsfragment for this one.

Reply:
Good catch Jed, missed that, thanks. I looked for a `28283` newsfragment, but weirdly the corresponding one is `08212`. :thinking: